web/src/components/__tests__/EventListItem.spec.tsx

```diff
@@ -20,7 +20,7 @@ describe('EventListItem', () => {
     const { getByText, getByRole } = renderWithRouterAndTheme(<EventListItem event={event} formatter={formatter} />)
 
     expect(getByText(event.title)).toBeTruthy()
-    expect(getByText(event.date.toFormattedString(formatter))).toBeTruthy()
+    expect(getByText('Mar 6, 2019, 11:04 PM - Jan 1, 2015, 4:10 AM')).toBeInTheDocument()
```

Comment:
The problem is that react testing library automatically normalizes whitespaces (see https://testing-library.com/docs/queries/about/#normalization)
Setting `collapseWhitespace: false` fixes the problem: 
```suggestion
    expect(getByText(event.date.toFormattedString(formatter), { collapseWhitespace: false })).toBeTruthy()
```